### ISSUE TITLE
Hello Madam I've An Issue 

### ISSUE THREAD

**Issue by cprabhat22** [samia17/Medical_Records_Miniproject#1]
Can We Discuss On It

**Comment by naheelsidd786** [comment]
Hello Madam can you please help with this project I really need it

**Comment by SamratEmily** [comment]
> Can We Discuss On It

Can you run this project?

**Comment by cprabhat22** [comment]
I've done the project and submitted too you've responded late sorry

**Comment by SamratEmily** [comment]
I need to run it to know how to run a blockchain project. Could you please help me?

**Comment by SamratEmily** [comment]
Please help me to run, I don't need to submit it. I only need to know how
to run a project based on solidity.

**Comment by cprabhat22** [comment]
I HAVE USED SOLIDITY (ON REMIX IDE) , GANACHE APPLICATION , METAMASK PLUGIN
AND CONNECTED THE SAME WITH MY HARDWARE IOT MEDICAL DOMAIN

IF YOU NEED I CAN EXPLAIN THE SAME ON GOOGLE MEET

**Comment by SamratEmily** [comment]
Yes please, it will help me a lot. I will be very much grateful to you. How
can we connect?

**Comment by cprabhat22** [comment]
Send me a meeting ID we'll join tommorow afternoon

**Comment by SamratEmily** [comment]
Meet id: https://meet.google.com/tce-iaao-dfy

I will wait for you at 3 pm BD. In an emergency, You can message me on
WhatsApp 01973249268.

**Comment by cprabhat22** [comment]
What's your location?

**Comment by SamratEmily** [comment]
Bangladesh ☺️, You can message me on WhatsApp.

**Comment by cprabhat22** [comment]
We'll join tommorow on meet just keep downloaded your ganache from truffle
suite website and metamask plugin

**Comment by SamratEmily** [comment]
Hey, I am waiting. If you are active, inform me delay of time.

**Comment by cprabhat22** [comment]
Join meeting

**Comment by SamratEmily** [comment]
Hey Bro, Is there any problem?
